cni/network/invoker_azure_test.go

```diff
@@ -127,15 +135,14 @@ func TestAzureIPAMInvoker_Add(t *testing.T) {
 		fields  fields
 		args    args
 		want    *cniTypesCurr.Result
-		want1   *cniTypesCurr.Result
 		wantErr bool
 	}{
 		{
 			name: "happy add ipv4",
 			fields: fields{
 				plugin: &mockDelegatePlugin{
 					add: add{
-						resultsIPv4: getResult("10.0.0.1/24"),
+						resultsIPv4: getSingleResult("10.0.0.1/24"),
```

Comment:
Can you move the subnet to a var at the beginning of the file?

Reply:
updated